### ISSUE TITLE
Stack trace being printed into status window when typing into thread

### ISSUE THREAD

**Issue by Nicd** [wee-slack/wee-slack#358]
There seems to be a problem, maybe related to threads and typing notifications. This stack trace was continually printed into the status window:

```
2017-04-21T16:17:08+0300 !! python: error in function "typing_notification_cb"
2017-04-21T16:17:08+0300 python: stdout/stderr: Traceback (most recent call last):
2017-04-21T16:17:08+0300 python: stdout/stderr:   File "/home/nicd/.weechat/python/wee_slack.py", line 695, in typing_notification_cb
2017-04-21T16:17:08+0300 python: stdout/stderr:     identifier = channel.identifier
2017-04-21T16:17:08+0300 python: stdout/stderr: AttributeError: 'SlackThreadChannel' object has no attribute 'identifier'
```

It happens whenever typing into a thread channel.

**Comment by dzfweb** [comment]
+1

**Comment by dzfweb** [comment]
fixed at #https://github.com/wee-slack/wee-slack/pull/386/commits/cdb0849dc432b186cf9d1efe1d31f07c4a09eedc

**Comment by trygveaa** [comment]
This has been fixed in master with #394.
